### PR TITLE
fix: use correct variable when logging BuildName

### LINKED ISSUE #132: Strange output from build and push command
When setting the `IMAGE_NAME` env variable I get output like:

```
Using <src-dir> as BuildNameUsing <src-dir> as BuildNameUsing <src-dir>as BuildNameUsing <src-dir> as BuildNamePushing tag 'xxx.dkr.ecr.eu-west-1.amazonaws.com/<IMAGE_NAME>:build'
```

### PR DESCRIPTION
**Pull request by argoyle**
Closes #132

### CODE REVIEW

File: pkg/ci/ci.go [hunk 1]
```diff
@@ -36,7 +36,7 @@ func (c *Common) SetImageName(imageName string) {
 
 func (c *Common) BuildName(name string) string {
 	if c.ImageName != "" {
-		fmt.Printf("Using %s as BuildName", name)
+		fmt.Printf("Using %s as BuildName\n", c.ImageName)
```
Comment (argoyle) [inline pkg/ci/ci.go]: Should we try to pass in the logger here instead of fmt.Printf'ing?

Comment (peter-svensson) [reply pkg/ci/ci.go]: Might be a good idea 👍

**Review by peter-svensson — APPROVED** [review]
Looks good!

**Review by peter-svensson — APPROVED** [review]
💪

**Pull request merged**